laws/src/main/scala/cats/laws/ApplicativeEvalLaws.scala

```diff
@@ -0,0 +1,24 @@
+package cats
+package laws
+
+import cats.Eval.{always, now}
+import cats.syntax.all._
+
+trait ApplicativeEvalLaws[F[_]] extends ApplicativeLaws[F] {
+  implicit override def F: ApplicativeEval[F]
+
```

Comment:
No, because the purpose of `ApplicativeEval` is evaluation in the `F[_]` context, but it doesn't necessarily have to be lazy, such that `Try` and `Future` can implement it.

What you're thinking of is the other type class called `MonadDefer`.